src/test/kotlin/ch/srgssr/pillarbox/monitoring/event/setup/AliasSetupTaskTest.kt

```diff
@@ -0,0 +1,84 @@
+package ch.srgssr.pillarbox.monitoring.event.setup
+
+import ch.srgssr.pillarbox.monitoring.event.repository.OpenSearchConfigurationProperties
+import ch.srgssr.pillarbox.monitoring.test.PillarboxMonitoringTestConfiguration
+import ch.srgssr.pillarbox.monitoring.test.createDispatcher
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.kotest.assertions.throwables.shouldNotThrow
+import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.matchers.shouldBe
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+
+@SpringBootTest
+@ContextConfiguration(classes = [PillarboxMonitoringTestConfiguration::class])
+@ActiveProfiles("test")
+class AliasSetupTaskTest(
+  private val aliasSetupTask: AliasSetupTask,
+  private val openSearchProperties: OpenSearchConfigurationProperties,
+  private val objectMapper: ObjectMapper,
+) : ShouldSpec({
+
+    var mockWebServer = MockWebServer()
+
+    beforeTest {
+      mockWebServer = MockWebServer()
+      mockWebServer.start(openSearchProperties.uri.port)
+    }
+
+    afterTest {
+      mockWebServer.shutdown()
+    }
+
+    should("should skip creation if alias exists") {
```

Comment:
The test name shouldn't start with `should` if you’re using the `should` method, should it? 😛 
```suggestion
    should("skip creation if alias exists") {
```

https://kotest.io/docs/framework/testing-styles.html#should-spec

Reply:
Indeed, It shouldn't should twice